gradle.properties

```diff
@@ -0,0 +1,4 @@
+android.enableJetifier=false
```

Comment:
Jetifier is not needed according to Build Analyzer

Reply:
Hm, `false` doesn't compile for me 🤔 Somehow Gradle thinks there are still legacy support libraries:

> WARNING:Your project has set `android.useAndroidX=true`, but configuration 
> `:app:debugRuntimeClasspath` still contains legacy support libraries,
> which may cause runtime issues.
>
> This behavior will not be allowed in Android Gradle plugin 8.0.                 
>
> Please use only AndroidX dependencies or set `android.enableJetifier=true` in the
> `gradle.properties` file to migrate your project to AndroidX (see 
> https://developer.android.com/jetpack/androidx/migrate for more info)

I just made another commit that sets it to `true` to make it compile. But I should have a deeper look at this…